src/test/java/com/epam/steps/common/UICheckSteps.java

```diff
@@ -41,11 +41,10 @@ public void seeAllElementsArePresentOnAdminPage() {
                 .withFailMessage("Elements in admin page are not present")
                 .isTrue();
     }
-
-    @Then("Check all elements are present on the chosen section")
-    public void seeAllElementsArePresentInStudentSection() {
-        assertThat(adminPage.checkUIofStudentsAndParentsSections())
-                .withFailMessage("Elements are not present on the chosen section")
+    @Then("Check all elements are present in the section")
```

Comment:
Please, write more clearable step name, for example - Check all elements are present in the chosen section

Reply:
done